### PR TITLE
macOS instead of MacOS

### PR DESCRIPTION
**Pull request by arialdomartini**
Since June 13, 2016 the name of the Apple's operating system has been changed to macOS. Both OS X and MacOS should considered wrong.

The Guidelines used to contain MacOS. This PR fixes the name to macOS.

(Yes, actually this is a pull request for 1 single modified character)

### CODE REVIEW

**Review by FlorianWendelborn — APPROVED** [review]

**Comment by RobDolinMS** [comment]
Thanks for this @arialdomartini.  I checked [Apple's website](https://www.apple.com/macos/sierra/) and they use "macOS" instead of "MacOS"

LGTM

Can I get a second LGTM from another maintainer?  /cc @garethj-msft @markdstafford @cleemullins

**Comment by cleemullins** [comment]
Lgtm.

Merged.

**Comment by 7mobile56** [comment]
鼎為鍊銀. 爐為鍊金。

**Pull request merged**
